### PR TITLE
Improve some issues of #18 

### LINKED ISSUE #18: Minor design issues
제목 없음

1. The block doesn't align with the grid. Tetris block should be moved down and right by the half of the grid line width.

2. The font of the speed and score are weird. It would be great if you match them to the font of the title.

3. The temporary storage circle and the next Tetris block circle should be different. It is difficult to recognize the difference while we are playing.

### PR DESCRIPTION
**Pull request by oh3vci**
I improved some issues of #18.

- Add google web font to document
- Add text to canvas for separating of the hold and the next block areas

### CODE REVIEW

File: css/screen.css [hunk 1]
```diff
@@ -15,6 +16,7 @@ canvas {
   left:0;
   top:0;
 }
+
```
Comment (DevSDK) [inline css/screen.css:19]: Nice clean up : )

File: components/game/block-preview.js [hunk 2]
```diff
@@ -42,6 +46,13 @@ class PreviewBlockManager {
       this.previewBlockArray[i].draw(ctx);
     }
   }
+
+  _drawTitle(ctx) {
```
Comment (DevSDK) [inline components/game/block-preview.js:50]: This seems repeated with ShiftBlock's. 
Perhaps we could make an abstraction between PreviewBlockManager and ShiftBlock.

I guess this function could be generalized for drawing text on the canvas. 
So I suggest that moving out of the classes to another location(like a utility? or global?) with renaming as `drawLabel` then receive parameters that are required information such as canvas-context, position, text, and so on.

Example:
```javascript
 drawLabel(ctx, 'NEXT', shiftBackgroundColor, '16px', previewBlockLocation[0][0], 15)
```

I personally think handling the context should be wrapped by a function for readability.

WDYT about this?

File: index.html [hunk 3]
```diff
@@ -35,8 +34,8 @@ <h5 id="game-over">GAME OVER</h5>
       <div class="screen">
         <img id="youngImg" src="res/young.png"/>
         <div id="stage">
-            <canvas id="canvas" width="500" height="680"></canvas>
-            <canvas id="background" width="500" height="680"></canvas>
+          <canvas id="canvas" width="500" height="680"></canvas>
```
Comment (DevSDK) [inline index.html:37]: Thanks for the correction of the indent!

**Review by DevSDK — COMMENTED** [review]
Thanks for the improvement! Nice refactor! I left a comment just nit consideration!

File: components/game/block-preview.js (hunk 2 above)
Comment (WoochanLee) [reply components/game/block-preview.js:50]: Agree.

I think it would be very good to refactoring it like that.

File: components/game/block.js [hunk 4]
```diff
@@ -150,7 +150,7 @@ class ControlBlock {
     ]);
   }
 
-  changeContorlBlock(controlBlockType) {
```
Comment (WoochanLee) [inline components/game/block.js:153]: Thanks ~~!

**Review by WoochanLee — APPROVED** [review]
LGTM!

I think it would be nice to have a pretty background.
It seems that Dae-ho Kim has a great ability in this area.

And the circle's border is a bit broken, so it seems that anti-aliasing is needed.

스크린샷 2021-03-26 오전 11 50 53

**Review by DevSDK — APPROVED** [review]

**Pull request merged**